google-cloud/sql/inputs.tf

```diff
@@ -87,3 +87,8 @@ variable "project" {
   description = "Name of gcp project"
   type        = "string"
 }
+
+variable "namespace" {
+  description = "Name of kubernetes namespace"
+  type        = "string"
```

Comment:
Default this to prometheus,, just like mysql scraper does.

Reply:
done